roles/dovecot/templates/etc/dovecot/conf.d/20-imap.conf

```
@@ -36,7 +36,11 @@
 
 # Override the IMAP CAPABILITY response. If the value begins with '+',
 # add the given capabilities on top of the defaults (e.g. +XFOO XBAR).
+{% if dovecot_virtual_enabled %}
+imap_capability = +XDOVECOT
+{% else %}
 #imap_capability =
+{% endif %}
```

I wouldn't make this dependent on `dovecot_virtual_enabled`, as it may have other uses as well. Instead, please make this templatable separately:

```suggestion
imap_capability = {{ dovecot_imap_capability }}
```

And then add `dovecot_imap_capability: ""` to the role defaults.